VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/models/participant-status-base.ts

```diff
@@ -0,0 +1,37 @@
+import { HostListener } from '@angular/core';
+import { ParticipantStatusUpdateService } from 'src/app/services/participant-status-update.service';
+import { EventType } from 'src/app/services/clients/api-client';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { ActivatedRoute } from '@angular/router';
+
+export abstract class ParticipantStatusBase {
+    constructor(
+        protected participantStatusUpdateService: ParticipantStatusUpdateService,
+        protected logger: Logger,
+        protected route: ActivatedRoute
+    ) {}
+
+    raiseNotSignedIn() {
+        try {
```

Comment:
this try-catch block won't work since the promise is already handling errors on line 23.

Reply:
there is not only promise in this block of code.